src/app/components/feed/FeedLastClusterizedAt.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { createFragmentContainer, graphql } from 'react-relay/compat';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import ScheduleSendIcon from '../../icons/schedule.svg';
+import styles from './FeedLastClusterizedAt.module.css';
+
+const FeedLastClusterizedAt = ({ feed }) => {
+  const message = feed.last_clusterized_at ? (
+    <FormattedDate
+      value={feed.last_clusterized_at}
+      month="short"
+      day="2-digit"
+      hour="2-digit"
+      minute="2-digit"
+    />
+  ) : (
+    <FormattedMessage
+      id="feedClusters.feedClusterizedTimePendingInitialUpdate"
```

Comment:
Please use the component name as the namespace for this ID - so, `feedLastClusterizedAt.feedClusterizedTimePendingInitialUpdate` instead of `feedClusters.feedClusterizedTimePendingInitialUpdate`.